framework/src/test/java/tests/RangeTest.java

```diff
@@ -265,6 +276,28 @@ public void testShortRange() {
         }
     }
 
+    @Test
+    public void testCharRange() {
+        Range.ignoreOverflow = false;
+        for (Range range : ranges) {
+            Range result = range.charRange();
+            for (long value : values) {
+                if (value < range.from + CHAR_WIDTH
+                        && value > range.to - CHAR_WIDTH
+                        && !result.isCharEverything()) {
```

Comment:
This test is inconsistent with the corresponding tests for byte and short.  Should this one be changed, or those?

Reply:
Addressed. Changed to be consistent with other tests.